src/utils/getEllipsisStyle.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from "styled-components";
+
+export function getEllipsisStyle(linesNumber?: number) {
+  if (linesNumber) {
+    return css`
+      display: -webkit-box;
+      -webkit-line-clamp: ${linesNumber};
+      -webkit-box-orient: vertical;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    `;
```

Comment:
Questo stile non funziona su Firefox, nè su ogni altro browser non basato su webkit.
In più forse starebbe meglio nella cartella styles che in utils.